docs/examples/index.rst

```diff
@@ -127,13 +116,13 @@ Sentiment Analysis
    sentiment_analysis/index
 
 
-Sequence Sampling
------------------
+Text Generation
+---------------
 
 .. container:: cards
 
    .. card::
-      :title: Sequence Generation with Sampling and Beam Search
+      :title: Inference with Beam Search Sampler and Sequence Sampler
```

Comment:
Inference with Beam Search Sampler and Sequence Sampler
-> 
Sequence Generation with Beam Search Sampler and Sequence Sampler